requirements.txt

```diff
@@ -9,3 +9,4 @@ pypandoc
 pytest-benchmark
 pytest-docker-pexpect
 twine
+xdg
```

Comment:
I don't think the addition of a new dependency is worth in this case.

How about sticking to what's been done so far in the project:

https://github.com/nvbn/thefuck/blob/841e3f9e13f52747c18319b55c77ece8095df306/thefuck/utils.py#L227-L228

In absence of `XDG_DATA_HOME`, Fish defaults to `~/.local/share`.

Reply:
Good point. I haven't written that much Python code so I kept sticking to a library. I've pushed a change.

Reply:
I understood that the `history` rule keeps suggesting `fuck (--yeah)`. If it's in shell's history and the `history` rule is suggesting it, then there might be an explanation for that. You can stick a breakpoint in the `match` function of `history` and inspect the input, it may help you understand that.